cli_reference/basic_cli_operations.adoc

```diff
@@ -14,7 +14,7 @@ This topic provides information on some general CLI operations and their syntax.
 You must link:get_started_cli.html[setup and login] with the CLI before you can
 perform these operations.
 
-== Common Operations
+== Common operations
```

Comment:
Are we no longer doing title case? And if not, I see you left title case in the table headings; intentional?